File: C99/1002.c
#include <stdio.h>
#include <math.h>
int main()
{
    double r,n,a;
    n=3.14159;
    scanf("%lf", &r);
    a=n*(pow(r,2));
    printf("A=%0.4lf\n",a);
    return 0;
}

File: Java14/1002.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
public class Main {
public static void main(String[] args) throws IOException {

    InputStreamReader ir = new InputStreamReader(System.in);
    BufferedReader in = new BufferedReader(ir);
    
	double pi=3.14159,inp,a;
	inp = Double.parseDouble(in.readLine());
		
	a = pi * Math.pow(inp,2);
	System.out.printf("A=%.4f\n",a);
		
	}
}